src/undocumented_names.jl

```diff
@@ -0,0 +1,26 @@
+"""
+    test_undocumented_names(module::Module)
+
+Test that all public names in `module` have a docstring (not including the module itself).
+
+!!! warning
+    For Julia versions before 1.11, this does not test anything.
```

Comment:
Maybe a bit clearer:
```suggestion
    When running this Aqua test in Julia versions before 1.11, it does nothing.
```
or even more explicit:
```suggestion
    When running this Aqua test in Julia versions before 1.11, it does nothing.
    Thus if you use continuous integration tests, make sure those are configured
    to use Julia >= 1.11 in order to benefit from this test.
```